shopinvader_schema_address/schemas.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import ExtendableModelMeta
+from pydantic import BaseModel
+
+from odoo.addons.pydantic import utils
+
+
+class Address(BaseModel, metaclass=ExtendableModelMeta):
+    id: int
+    name: str | None
+    street: str | None
+    street2: str | None
+    zip: str | None
+    city: str | None
+    phone: str | None
+    email: str | None
+    state: int | None
+    country: int | None
+
+    class Config:
+        orm_mode = True
+        getter_dict = utils.GenericOdooGetter
+
+    @classmethod
+    def from_res_partner(cls, odoo_rec):
```

Comment:
@marielejeune I changed from_orm to from_res_partner since in this case, the odoo_rec will always be a res_partner. You may need to change this in your module

Reply:
Indeed, I'll do that. Thanks